ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -150,11 +150,12 @@ define([
         deserialize: function () {
             if (this.propertyModel) {
                 var filter = this.propertyModel.get('value');
+                var filterValue = typeof(filter.value) === 'object' ? filter.value.value : filter.value;
                 switch (filter.type) {
                     // these cases are for when the model matches the filter model
                     case 'DWITHIN':
-                        if (CQLUtils.isPointRadiusFilter(filter)){
```

Comment:
❓  Would it be worthwhile to make this function do the `typeof(filter.value) === 'object' ? filter.value.value : filter.value` logic as well, so that we can continue to use that here, and so that it won't break for anything else that happens to be using it?

What causes this to sometimes be `filter.value` and other times `filter.value.value` ? I see from the INTERSECTS case this isn't the only time we have had to deal with this.

Reply:
* I can add the check to CQLUtils.isPointRadiusFilter()
* it's `filter.value.value` when a template is used. I do not know why it was designed that way